io/io/inc/TMemFile.h

```diff
@@ -23,7 +23,7 @@ class TMemFile : public TFile {
    struct ExternalDataRange_t {
       const char *fStart;
       const size_t fSize;
-      ExternalDataRange_t(const char * start, const size_t size) : fStart(start), fSize(size) {}
+      explicit ExternalDataRange_t(const char * start, const size_t size) : fStart(start), fSize(size) {}
```

Comment:
Should we also use a different name to improve the self description of the purpose (which is, if I understood correct, saying "please TMemFile, just use this memory as is, don't waste time coying, I promise I won't change it and also don't delete it, it's mine :) "

Reply:
Can you propose something?

Reply:
ExternalReadOnlyDataRange_t ?

Reply:
The thing that copies can also be read only. At some point I was considering ExternalCStyleMemRegion or ExternalNonOwned... something similar.

Reply:
ExternalPleaseUseWithoutCopyDataRange_t ...  now we just need to summarize that a bit :)

Reply:
`ZerocopyView_t`?

Reply:
Sounds good.